lib/msf/core/exploit/http/client.rb

```diff
@@ -22,6 +22,7 @@ def initialize(info = {})
 
     register_options(
```

Comment:
```
if framework.features.enabled?("feature_name")
  register_options([OPT::RHOST_URL])
end
```

Reply:
Would you mind pulling the latest of the https://github.com/rapid7/metasploit-framework/pull/13833 PR to get the latest features API